### ISSUE TITLE
follow-up to SPyder 3 error message after load data -> execute command

### ISSUE THREAD

**Issue by dwaynepaschall** [spyder-ide/spyder#5949]
## Description
After trying to load a csv file via the gui loader, I canceled out of that and attempted to execute a command from the editor (F9), which generated the following error. This was *After* the gui data file loader was canceled. 

This is a follow-on to the previously reported error

**What steps will reproduce the problem?**

1. (attempt to load file via gui data loader...this may or may not be a contributing factor)
2. import pandas as pd
import keras

3. layers = [LSTM(2), Dense(1)]

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\ddp\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 263, in <lambda>
    self.finished(ec, es))
  File "C:\Users\ddp\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 328, in finished
    self.show_data(justanalyzed=True)
  File "C:\Users\ddp\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 349, in show_data
    self.datatree.load_data(self.DATAPATH)
  File "C:\Users\ddp\Anaconda3\lib\site-packages\spyder_profiler\widgets\profilergui.py", line 461, in load_data
    stats_indi = [pstats.Stats(profdatafile),]
  File "C:\Users\ddp\Anaconda3\lib\pstats.py", line 72, in __init__
    self.init(arg)
  File "C:\Users\ddp\Anaconda3\lib\pstats.py", line 86, in init
    self.load_stats(arg)
  File "C:\Users\ddp\Anaconda3\lib\pstats.py", line 99, in load_stats
    with open(arg, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\ddp\\.spyder-py3\\profiler.results'

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
This error will be fixed in our 3.2.5 version, to be released in the next few days.

**Comment by dwaynepaschall** [comment]
Excellent news! Thanks so much
